impl/src/jvmTest/kotlin/org/jetbrains/kotlinx/dl/impl/preprocessing/PreprocessingFinalShapeTest.kt

```diff
@@ -47,7 +47,7 @@ class PreprocessingFinalShapeTest {
                 bottom = 7
             }
             .toFloatArray { }
-        assertEquals(TensorShape(186, 188, 3), preprocess.getOutputShape(TensorShape(200, 200, 3)))
+        assertEquals(TensorShape(178, 186, 3), preprocess.getOutputShape(TensorShape(190, 200, 3)))
```

Comment:
could you please explain these numbers changes?

Reply:
The number changed because dimension order changed. Before we used width, height, channels order, now height, width, channels. I also changed the input image height from 200 to 190 so the image would not be square. I figure it's better for testing.